closure/bin/calcdeps.py

```diff
@@ -43,9 +43,8 @@
 import sys
 
 
-_BASE_REGEX_STRING = '^\s*goog\.%s\(\s*[\'"](.+)[\'"]\s*\)'
-req_regex = re.compile(_BASE_REGEX_STRING % 'require')
-prov_regex = re.compile(_BASE_REGEX_STRING % 'provide')
+req_regex = re.compile('^(?:[^=]+=)?\s*goog\.require\(\s*[\'"](.+)[\'"]\s*\)')
+prov_regex = re.compile('^\s*goog\.(?:provide|module)\(\s*[\'"](.+)[\'"]\s*\)')
```

Comment:
I'm concerned about lack of external unit test on this change. It appears we have one internally, however. Because you signed the CLA, I can try pulling this internally and testing it.